<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ENV_SensorDlg</class>
 <widget class="QDialog" name="ENV_SensorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>传感器一</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temLcd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="humLcd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>传感器二</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temLcd_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="humLcd_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>传感器三</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temLcd_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="humLcd_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>传感器四</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temLcd_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="humLcd_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>3</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>门禁</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>门禁一</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>门禁二</string>
    </property>
   </widget>
   <widget class="QLabel" name="door">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>未启用</string>
    </property>
   </widget>
   <widget class="QLabel" name="door_2">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>未启用</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>水浸、烟雾</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>水浸</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>烟雾</string>
    </property>
   </widget>
   <widget class="QLabel" name="water">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>未启用</string>
    </property>
   </widget>
   <widget class="QLabel" name="smoke">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>未启用</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="quitBtn">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>540</y>
     <width>110</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>540</y>
     <width>110</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>环境设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitBtn</sender>
   <signal>clicked()</signal>
   <receiver>ENV_SensorDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
